src/containers/preview-modal.jsx

```diff
@@ -41,7 +41,7 @@ class PreviewModal extends React.Component {
     }
     render () {
         return (supportedBrowser() ?
-            <PreviewModalComponent
+            !this.props.hideIntro && <PreviewModalComponent
```

Comment:
It's not clear what happens here if `this.props.hideIntro` is true...

I think it would be cleaner to add a function elsewhere in this file that conditionally returns `<PreviewModalComponent.../>` or `null` based on `this.props.hideIntro`, and call that function here instead